core/src/main/kotlin/com/malinskiy/marathon/report/junit/TestInfoGenerator.kt

```diff
@@ -0,0 +1,66 @@
+package com.malinskiy.marathon.report.junit
+
+import com.malinskiy.marathon.analytics.internal.sub.TestEvent
+import com.malinskiy.marathon.execution.TestResult
+import com.malinskiy.marathon.execution.TestStatus
+import com.malinskiy.marathon.report.junit.model.JUnitReport
+import com.malinskiy.marathon.report.junit.model.Pool
+import com.malinskiy.marathon.report.junit.model.StackTraceElement
+import com.malinskiy.marathon.report.junit.model.TestCaseData
+import com.malinskiy.marathon.report.junit.model.TestSuiteData
+import java.text.SimpleDateFormat
+import java.util.*
+
+private val FORMATTER = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.US).apply {
+    timeZone = TimeZone.getTimeZone("UTC")
+}
+
+internal class JunitReportGenerator(private val testEvents: List<TestEvent>) {
+
+    @Suppress("MagicNumber")
+    private fun Long.toJUnitSeconds(): String = (this / 1000.0).toString()
```

Comment:
TimeUnit.MILLISECONDS.toSeconds(this)

Reply:
But this will round of the seconds and I am not sure it is the expected behaviour.
 '35.757' would become '35'.